types/types.ts

```diff
@@ -9,6 +9,7 @@ export interface WerkitSuccessResult<ResultType> extends WerkitMeta {
   result: ResultType
   error: null
   error_origin: null
+  messageKey : any
```

Comment:
Should this property messageKey be made optional to make it easier to migrate existing consumers of WerkitResult?

Reply:
I was thinking this could go into `result` but in that case there's no way to track where errors come from. Is that what you're thinking?

The publishers publish their own types (and declare dependencies on specific versions of werkit) so I don't think making it optional makes it any easier. Also I think `undefined` will match `any`.

Could you make it `message_key` to match the rest of these keys?

Reply:
> I was thinking this could go into result but in that case there's no way to track where errors come from. Is that what you're thinking?

Yes

> Could you make it message_key to match the rest of these keys?

Yes, will do.

Reply:
This is done.